Some bleeding tweaks

comma opened the pull request:
Small fraction of chemicals in blood leaks too, depending on strength of bleeding and volume of chems.
Bandaging arterial bleeder actually does something, it counts as putting pressure on it, so less bleeding.

comma committed:
Makes bandaging arterial bleeding matter

It now counts as putting pressure on it, lowering amount of blood lost.
Also keeps floors cleaner.

comma committed:
Chemicals bleed out too now

A small fraction of chemicals bleeds out with blood now, depending on how much chems is present and how big bleeding is.

PsiOmegaDelta commented on code/modules/organs/blood.dm:
```diff
@@ -38,6 +38,9 @@
 	if(remove_blood(amt))
+		if(bloodstr.total_volume)
+			var/chem_share = round(0.3 * amt * (bloodstr.total_volume/species.blood_volume), 0.01)
+			bloodstr.remove_any(chem_share * bloodstr.total_volume)
```
Just to check, but wouldn't this remove some more blood as well?

Techhead0 replied:
Blood is in `vessel`, this is `bloodstr`. IDK why they're separate.

That said... would it be better to handle this in `remove_blood()`? Also, the math seems vaguely funky to me.

PurpleMartinJCK replied:
Would remove blood then work with syringes?

comma replied:
Things that call remove_blood generally expect to get blood, so that might break things.
Math is using an empirically found constant (by which I mean I just tweaked it around until it was between 'unnoticeable' and 'might as well not bother with chems if you have a papercut' levels.

Techhead0 replied:
I didn't mean the 0.3, I meant the variables.

By my reckoning, it should vary with blood lost (`amt`) and chem concentration in blood.
Which I *think* is `bloodstr.total_volume/vessel.total_volume`.

comma replied:
Hm, yeah can go with current blood volume, I was checking against max possible one.

PsiOmegaDelta replied:
Things expecting to find only blood in either `vessel` or `bloodstr` have the wrong expectations in my opinion.

PurpleMartinJCK commented on code/modules/organs/internal/heart.dm:
```diff
@@ -128,7 +128,7 @@
 
 			if(temp.status & ORGAN_ARTERY_CUT)
-				var/bleed_amount = Floor((owner.vessel.total_volume / (temp.applied_pressure ? 400 : 250))*temp.arterial_bleed_severity)
+				var/bleed_amount = Floor((owner.vessel.total_volume / (temp.applied_pressure || !open_wound ? 400 : 250))*temp.arterial_bleed_severity)
```
Where does the blood go if they have a cut artery and no open wound? Should probably flood the cavity or something.

comma replied:
We don't simulate that stuff.

PurpleMartinJCK replied:
Yeah but unlike applied pressure which would hold the artery closed, not having an open wound would cause it to bleed into the cavity or something instead. Using a bandage to stop arterial bleeding kind of defeats the purpose doesn't it?

comma replied:
It doesn't stop it, it just making it bit less strong. Pushing at it with your hands work, why not a bandage?

comma committed:
Changed chem bleedout math a bit

Now uses current blood amount to check how much we should lose isntead of total possible max
